dtbase/webapp/app/sensors/templates/sensors.html

```diff
@@ -68,10 +68,54 @@ <h3>Choose sensors and time period</h3>
             </div>
 
           </div>
-          <button style="margin-top: 5px" type="button" onclick="requestTimeSeries(false)">Plot</button>
-          <button style="margin-top: 5px" type="button" onclick="requestTimeSeries(true)">Download</button>
```

Comment:
The Download doesn't seem to work yet.

Reply:
Good point - I think this is functionality worth having - implemented it now.